deployment/smp-server-linode.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# <UDF name="enable_store_log" label="Enable store log - enable saving SMP queues to append only log, and restoring them when the server is started." default="on" oneof="on, off" />
+# <UDF name="api_token" label="API Token - enable StackScript to create tags address and hash for SMP server domain/ip address and transport key hash from inside linode. Use "address:port#hash" as SMP server URL. Note: minimal permissions token should have are the following: Account - read/write, Linodes - read/write" default="" />
+# <UDF name="domain_address" label="Domain address" default="" />
+
+# log all stdout output to stackscript.log
+exec &> >(tee -i /var/log/stackscript.log)
+# enable debugging features
+set -xeo pipefail
+
+cd $HOME
+
+# download latest release
+curl -s https://api.github.com/repos/simplex-chat/simplexmq/releases/latest \
+| grep "browser_download_url.*20_04-x86-64*" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+
+# move smp-server executable to /opt/simplex/bin
+mv smp-server-ubuntu-20_04-x86-64 smp-server
+chmod +x smp-server
+mkdir -p /opt/simplex/bin
+mv ./smp-server /opt/simplex/bin
```

Comment:
`mkdir` first and do only one move:
```
mkdir -p /opt/simplex/bin
mv smp-server-ubuntu-20_04-x86-64 /opt/simplex/bin/smp-server
chmod +x /opt/simplex/bin/smp-server
```

also why in the second move it's `./smp-server` with dot?

Reply:
done
just referenced current directory